README.md

```diff
@@ -1,23 +1,28 @@
-# Files
+## Files
 
 The actual documentation files live in the `/source/includes` directory.
 
-# Deployment Process
+## Running Slate
 
-First, run this command to build the docs:
+If you wish to run the development server for Slate to aid in working on the site, run:
 
 ```
-docker run --rm --name slate -v $(pwd)/build:/srv/slate/build -v $(pwd)/source:/srv/slate/source slatedocs/slate
+docker run --rm --name slate -p 4567:4567 -v $(pwd)/source:/srv/slate/source slatedocs/slate serve
 ```
 
-Then, run this command to deploy them:
+and you will be able to access your site at http://localhost:4567 until you stop the running container process.
+
+## Deployment Process
+
+First, run this command to build the docs:
 
 ```
-./deploy.sh --push-only
+docker run --rm --name slate -v $(pwd)/build:/srv/slate/build -v $(pwd)/source:/srv/slate/source slatedocs/slate
 ```
 
```

Comment:
```suggestion
After building the docs you will need to deploy the changes. To do that you run this command:

```./deploy.sh --push-only```

Reply:
The docs are manually deployed right now - which is why you still have to do this step.

Reply:
Oh this is a bad merge, thanks for catching this.